pkgs/top-level/python-packages.nix

```diff
@@ -11247,14 +11247,13 @@ in {
   linode-api = callPackage ../development/python-modules/linode-api { };
 
   livereload = buildPythonPackage rec {
```

Comment:
could you move this expression to `python-modules/livereload/default.nix`

Reply:
✅